From 38e7f38824752de2988be170bd500b65916a51c4 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 4 Nov 2019 18:06:44 +0000
Subject: [PATCH 0899/1000] MANTIS 0056315 bugfix: ConveyorSBD: AEA interface
 not initialized

platform.idl
V4.0

- set/user prepareToken in prepareDevice; activeToken is not set in prepareDevice


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1495 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index b52e6b26..c65ed4ce 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "3.9";
+    const string interface_version = "4.0";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -331,6 +331,10 @@ module PlatformManagement
         /** Gets the currently active token.
             @return the currently active token. */
         string getActiveToken();
+
+        /** Gets the activation token
+            @return the token of the application to be activated. */
+        string getPrepareToken();
         
         /** Dispatches a platform event.
             @param event - The plaform event
-- 
2.41.0.windows.1

